.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+**Description of changes:**
+
+**Related Issues:**
+
+Linked Issues:
+- Issue #
+- Issue #
+
+Issues closed by this PR:
+- Closes #
+- Closes #
+
+**Before merging:**
+
+- [ ] Did the CI tests pass?
```

Comment:
Given we require CI to pass before merging, probably don't need the extra note here?